docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,94 @@
+# Using a coding standard
+
+Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories. You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
```

Comment:
Here, I'm using "tool and code pattern configurations" as a compound noun, where "configurations" is the main word:

(tool and code pattern) + **configurations**

So, the rule for plurals of multi-word compound nouns applies (see [Rule 2 here](https://findanyanswer.com/goto/92732) or [my try at explaining this](https://docs.google.com/document/d/1CTL0mUXp-iQv-PX8YhQBj94YkrguLqlOwmZENNPnukY/edit?disco=AAAAIsoMTt8) :nerd_face: ).

Similarly, this is why we say for instance "coffee shops" and not "coffees shops", or "bus stops" and not "buses stops". Although I agree that since this is an unusual word combination, it may sound awkward.

Reply:
```suggestion
Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories.
```

We are going to release without the default 
https://docs.google.com/document/d/1UBXdsITNEf380UhjwckKaxT6z1ubANTQQwip0QG17Nk/edit?disco=AAAARRx3SIM